roles/rciam-metrics/templates/fastapi/fastapi.service.j2

```diff
@@ -5,11 +5,11 @@ After=network.target
 [Service]
 User={{ metrics_user.name }}
 Group=www-data
-
+Restart=always
 WorkingDirectory={{ metrics_path }}
 Environment="PATH={{ metrics_path }}/.venv/bin"
-ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.main:app --log-syslog --log-syslog-prefix {{ metrics_api.log_identifier }}-service --bind {{ lookup('dig', inventory_hostname) }} --log-level {{ metrics_api.log_level }}
-
+ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 8 -k uvicorn.workers.UvicornWorker app.main:app --bind 0.0.0.0:8000 --log-level {{ metrics_api.log_level }} --timeout {{ metrics_api.timeout }}
```

Comment:
Check if we need to bind to all interfaces or we can just bind to localhost. This depends on the vhost configuration in nginx